<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1109</width>
    <height>874</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="graphHolder1" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>471</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="graphHolder2" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>471</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="graphHolder3" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>570</y>
      <width>471</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="startWaveSpinBox">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>80</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>1465.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1570.000000000000000</double>
    </property>
    <property name="value">
     <double>1515.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>80</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Starting Wavelength (nm)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>10</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Scan Parameters</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="endWaveSpinBox">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>120</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>1465.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1570.000000000000000</double>
    </property>
    <property name="value">
     <double>1525.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>120</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ending Wavelength (nm)</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="scanRateSpinBox">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>160</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>0.002400000000000</double>
    </property>
    <property name="maximum">
     <double>10.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>160</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Scan Rate (nm/s)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>200</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sampling Rate (samples/s)</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sampleRateSpinBox">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>200</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>50000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1000.000000000000000</double>
    </property>
    <property name="value">
     <double>1000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLineEdit" name="fileNameLineEdit">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>330</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>test</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>767</x>
      <y>330</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>File name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startScanButton">
    <property name="geometry">
     <rect>
      <x>907</x>
      <y>520</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start Scan</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>717</x>
      <y>240</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Inputs (each one &lt;br&gt;on new line)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>767</x>
      <y>370</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Metadata&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="graphChoices3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>570</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="metadataTextEdit">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>370</y>
      <width>231</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="inputsTextEdit">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>240</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="plainText">
     <string>ai0: Gas Cell
ai1: Interferometer
ai2: Reference</string>
    </property>
   </widget>
   <widget class="QComboBox" name="graphChoices2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>290</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="graphChoices1">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>663</x>
      <y>10</y>
      <width>20</width>
      <height>811</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <zorder>graphHolder2</zorder>
   <zorder>graphHolder3</zorder>
   <zorder>startWaveSpinBox</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>endWaveSpinBox</zorder>
   <zorder>label_3</zorder>
   <zorder>scanRateSpinBox</zorder>
   <zorder>label_4</zorder>
   <zorder>graphHolder1</zorder>
   <zorder>label_5</zorder>
   <zorder>sampleRateSpinBox</zorder>
   <zorder>fileNameLineEdit</zorder>
   <zorder>label_6</zorder>
   <zorder>startScanButton</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>graphChoices3</zorder>
   <zorder>metadataTextEdit</zorder>
   <zorder>inputsTextEdit</zorder>
   <zorder>graphChoices2</zorder>
   <zorder>graphChoices1</zorder>
   <zorder>line</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1109</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
